src/assets/i18n/en.json

```diff
@@ -23,5 +27,19 @@
     },
     "PATIENT_SEARCH": {
         "LOADING": "Loading"
+    },
+    "DIALOG": {
+        "TITLE": "Parient Register",
+        "CANCEL": "Cancel",
+        "REGISTER": "Register"
```

Comment:
DIALOG object should be inside PATIENT_CREATE as it's a part of the creation process

Reply:
done